miden/tests/integration/operations/u32_ops/arithmetic_ops.rs

```diff
@@ -703,82 +427,8 @@ fn u32overflowing_madd() {
 }
 
 #[test]
-fn u32checked_div() {
-    let asm_op = "u32checked_div";
-
-    test_div(asm_op);
-}
-
-#[test]
-fn u32checked_div_fail() {
-    let asm_op = "u32checked_div";
-
-    // should fail if a >= 2^32
-    let test = build_op_test!(asm_op, &[U32_BOUND, 1]);
-    test.expect_error(TestError::ExecutionError("NotU32Value"));
-
-    // should fail if b >= 2^32
-    let test = build_op_test!(asm_op, &[1, U32_BOUND]);
-    test.expect_error(TestError::ExecutionError("NotU32Value"));
-
-    // should fail if b == 0
-    let test = build_op_test!(asm_op, &[1, 0]);
-    test.expect_error(TestError::ExecutionError("DivideByZero"));
-}
-
-#[test]
-fn u32checked_div_b() {
-    let build_asm_op = |param: u32| format!("u32checked_div.{param}");
-
-    // --- simple cases ---------------------------------------------------------------------------
-    let test = build_op_test!(build_asm_op(1).as_str(), &[0]);
-    test.expect_stack(&[0]);
-
-    // division with no remainder
-    let test = build_op_test!(build_asm_op(1).as_str(), &[2]);
-    test.expect_stack(&[2]);
-
-    // division with remainder
-    let test = build_op_test!(build_asm_op(2).as_str(), &[1]);
-    test.expect_stack(&[0]);
-
-    // --- random u32 values ----------------------------------------------------------------------
-    let a = rand_value::<u32>();
-    let mut b = rand_value::<u32>();
-    if b == 0 {
-        // ensure we're not using a failure case.
-        b += 1;
-    }
-    let expected = (a / b) as u64;
-
-    let test = build_op_test!(build_asm_op(b).as_str(), &[a as u64]);
-    test.expect_stack(&[expected]);
-
-    // --- test that the rest of the stack isn't affected -----------------------------------------
-    let c = rand_value::<u64>();
-    let test = build_op_test!(build_asm_op(b).as_str(), &[c, a as u64]);
-    test.expect_stack(&[expected, c]);
-}
-
-#[test]
-fn u32checked_div_b_fail() {
-    let build_asm_op = |param: u64| format!("u32checked_div.{param}");
-
-    // should fail during execution if a >= 2^32.
-    let test = build_op_test!(build_asm_op(1).as_str(), &[U32_BOUND]);
-    test.expect_error(TestError::ExecutionError("NotU32Value"));
-
-    // should fail during compilation if b >= 2^32.
-    test_param_out_of_bounds("u32checked_div", U32_BOUND);
-
-    // should fail during compilation if b = 0.
-    let test = build_op_test!(build_asm_op(0).as_str());
-    test.expect_error(TestError::AssemblyError("division by zero"));
-}
-
-#[test]
-fn u32unchecked_div() {
-    let asm_op = "u32unchecked_div";
+fn u32div() {
+    let asm_op = "u32div";
 
     // should push d = (a * b) / 2^32 onto the stack.
     test_div(asm_op);
```

Comment:
Do we actually need `test_div`, `test_mod`, and `test_divmod` helper functions now? Seems like they are called only from one place and maybe it makes sense to move the contents of the functions to their call sites?